frontend/src/components/Analysis/utils.tsx

```diff
@@ -76,12 +72,22 @@ export function parse(
 
   // Sort by the correct bucket interval order
   bucketPerformances.sort((a, b) => {
-    if (a.bucket_interval[0] > b.bucket_interval[0]) {
-      return 1;
-    } else if (a.bucket_interval[0] < b.bucket_interval[0]) {
-      return -1;
+    if (a.bucket_name !== undefined) {
```

Comment:
Let's use `!= null` to exclude `null` and `undefined`.

Reply:
Good catch, thanks!